src/apps/common/ns_turn_utils.h

```diff
@@ -45,7 +50,8 @@ extern "C" {
 //////////////////////// LOG //////////////////////////
 
 typedef enum {
-  TURN_LOG_LEVEL_INFO = 0,
+	TURN_LOG_LEVEL_DEBUG = 0,
```

Comment:
Do not want to dwell on it too much (I would remove all logging changes unrelated to windows) but if we change the logging level I would suggest to adopt standard de-facto: https://logging.apache.org/log4j/2.x/manual/customloglevels.html